boote/forms.py

```diff
@@ -50,239 +50,263 @@
 DURATION.append(["150", "2.5 Stunden"])
 DURATION.append(["180", "3 Stunden"])
 
-
 BOOKING_TYPE = []
-BOOKING_TYPE.append(['AUS', 'Ausbildung'])
-BOOKING_TYPE.append(['REG', 'Regatta'])
-
-MONTHS= []
-MONTHS.append(['01', 'Januar'])
-MONTHS.append(['02', 'Februar'])
-MONTHS.append(['03', 'Maerz'])
-MONTHS.append(['04', 'April'])
-MONTHS.append(['05', 'Mai'])
-MONTHS.append(['06', 'Juni'])
-MONTHS.append(['07', 'Juli'])
-MONTHS.append(['08', 'August'])
-MONTHS.append(['09', 'September'])
-MONTHS.append(['10', 'Oktober'])
-MONTHS.append(['11', 'November'])
-MONTHS.append(['12', 'Dezember'])
+BOOKING_TYPE.append(["AUS", "Ausbildung"])
+BOOKING_TYPE.append(["REG", "Regatta"])
+
+MONTHS = []
+MONTHS.append(["01", "Januar"])
+MONTHS.append(["02", "Februar"])
+MONTHS.append(["03", "Maerz"])
+MONTHS.append(["04", "April"])
+MONTHS.append(["05", "Mai"])
+MONTHS.append(["06", "Juni"])
+MONTHS.append(["07", "Juli"])
+MONTHS.append(["08", "August"])
+MONTHS.append(["09", "September"])
+MONTHS.append(["10", "Oktober"])
+MONTHS.append(["11", "November"])
+MONTHS.append(["12", "Dezember"])
 
 DAYS = []
 for i in range(1, 10):
```

Comment:
Merge both ranges to one and use DAYS.append([f"{i:02d}", f"{i:02d}"]) to get the leading zeros.

Reply:
I agree that the initialization code could be improved, but I'll leave this up to another PR. Instead of using hard coded values we should consider using a Datepicker, which would prohibit selecting dates like 31. February.